### ISSUE TITLE
How would I go about styling the appearance of a custom component to look like a UI Kitten component? 

### ISSUE THREAD

**Issue by Caperious** [akveo/react-native-ui-kitten#1344]
## 💬 Question

More specifically I'm creating a time picker component and would like to know if I can access the default paddings, margins, etc..

I want to style it, to resemble the date picker component as closely as possible. 

## UI Kitten and Eva version

| Package      | Version |
| ----------- | ----------- |
| @eva-design/eva      |   2.0.0     |
| @ui-kitten/components   |    5.0.0     |

**Comment by artyorsh** [comment]
👋 
We have a request for this component, but still no "official" implementation. You can check [the comment](https://github.com/akveo/react-native-ui-kitten/issues/778#issuecomment-630447282) I posted to the related thread. Does it cover your needs?

> default paddings, margins, etc

There is no default spacings defined in Eva unfortunately. However, you may see the list of theme variables [here](https://akveo.github.io/react-native-ui-kitten/docs/design-system/eva-light-theme) and read how to use it [here](https://akveo.github.io/react-native-ui-kitten/docs/design-system/use-theme-variables).

**Comment by Caperious** [comment]
I know, I made a workaround using [react-native-modal-datetime-picker](https://github.com/mmazzarolo/react-native-modal-datetime-picker) and styling a button to look kinda like an input, with hardcoded values. I used the theme variables to apply the color styles and fonts.

I think it would be beneficial to also expose the margins, paddings, and other properties, to be able to consistently style the application. I'm guessing there will be some components that will have to be custom made, but the look and feel should fit in the design system. 

Otherwise great job on the framework, I like it a lot.
